neuronunit/tests/fi.py

```diff
@@ -83,7 +83,7 @@ def f(ampl):
                 uc = {'amplitude':ampl}
                 current.update(uc)
                 model.inject_square_current(current)
-                n_spikes = model.get_spike_count()
+                n_spikes = model._backend.get_spike_count()
```

Comment:
@russelljjarvis 
Again, this should work with `model.get_spike_count`.  If it doesn't, then something needs to be fixed in `model.get_spike_count`.